src/renderer/widgets/AddProxyModal/ui/AddProxyForm.tsx

```diff
@@ -144,11 +145,13 @@ const SignatorySelector = () => {
 
   if (!isMultisig) return null;
 
-  const options = signatories.map(({ signer, balance }) => {
+  const options = signatories.reduce<DropdownOption[]>((acc, { signer, balance }) => {
+    if (!signer?.id) return acc;
```

Comment:
What kind of problem is solved here?

Reply:
https://github.com/novasamatech/nova-spektr/issues/1614